source/operations/checklists/software.rst

```diff
@@ -8,4 +8,62 @@ Software Checklist
    :local:
    :depth: 2
 
-Stub - detail software checklist items
+Use the following checklist when planning the software configuration for a production, distributed MinIO deployment.
+
+Pre-MinIO Install Requirements
+------------------------------
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Servers running a Linux operating system, such as a supported version of Red Hat Enterprise Linux (RHEL) or Ubuntu LTS
+
+   * - :octicon:`circle` 
+     - System administrator access to the remote servers
+
+   * - :octicon:`circle`
+     - A distributed deployment solution, such as `Podman <https://podman.io>`__ for containers or `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`__ for Kubernetes
+
+   * - :octicon:`circle`
+     - Load balancer to handle routing of requests (for example, `NGINX <https://www.nginx.com/>`__)
+
+   * - :octicon:`circle`
+     - :ref:`Prometheus / Grafana <minio-metrics-collect-using-prometheus>` setup
+
+   * - :octicon:`circle`
+     - :ref:`Security Token Service (STS) <minio-external-identity-management-ad-ldap>` defined
+
+   * - :octicon:`circle` 
+     - (optional) :mc:`mc` installed on the local host system
+
+
+MinIO Install
+-------------
+
+See the :ref:`Multi Node Multi Drive deployment guide <deploy-minio-distributed>` and the :ref:`Minio Tenant deployment guide <minio-k8s-deploy-minio-tenant>`.
+
+
+Post Install Tasks
```

Comment:
Lets try to flesh out why someone would want to do any of these.

Reply:
Added more info to each checklist item